climpred/metrics.py

```diff
@@ -126,14 +134,93 @@ def _mae(forecast, reference, dim='svd', comparison=None):
+def _brier_score(forecast, reference, **kwargs):
+    """Calculate Brier score for forecasts on binary reference.
+
+    ..math:
+        BS(f, o) = (f - o)^2
+
+    Args:
+        * forecast (xr.object)
+        * reference (xr.object)
+        * func (function): function to be applied to reference and forecasts
+                           and then mean('member') to get forecasts and
+                           reference in interval [0,1].
+                           (required to be added via **kwargs)
+
+    Reference:
+        * Brier, Glenn W. “VERIFICATION OF FORECASTS EXPRESSED IN TERMS OF
+        PROBABILITY.” Monthly Weather Review 78, no. 1 (1950).
+        https://doi.org/10.1175/1520-0493(1950)078<0001:VOFEIT>2.0.CO;2.
+
+    Example:
+        >>> def pos(x): return x > 0
+        >>> compute_perfect_model(ds, control, metric='brier_score', func=pos)
+
+    See also:
+        * properscoring.brier_score
+        * xskillscore.brier_score
+    """
+    if 'func' in kwargs:
+        func = kwargs['func']
+    else:
+        raise ValueError(
+            'Please provide a function `func` to be applied to comparison and \
+             reference to get values in  interval [0,1]; \
+             see properscoring.brier_score.'
+        )
+    return brier_score(func(reference), func(forecast).mean('member'))
+
+
+def _threshold_brier_score(forecast, reference, **kwargs):
+    """
+    Calculate the Brier scores of an ensemble for exceeding given thresholds.
+    Provide threshold via kwargs.
+
+    .. math::
+        CRPS(F, x) = \int_z BS(F(z), H(z - x)) dz
+
+    Range:
+        * perfect: 0
+        * min: 0
+        * max: 1
+
+    Args:
+        * forecast (xr.object)
+        * reference (xr.object)
+        * threshold (int, float, xr.object): Threshold to check exceedance,
+            see properscoring.threshold_brier_score
+            (required to be added via **kwargs)
+
+    References:
+        * Brier, Glenn W. “VERIFICATION OF FORECASTS EXPRESSED IN TERMS OF
+        PROBABILITY.” Monthly Weather Review 78, no. 1 (1950).
+        https://doi.org/10.1175/1520-0493(1950)078<0001:VOFEIT>2.0.CO;2.
+
+    Example:
+        >>> compute_perfect_model(ds, control,
+                                  metric='threshold_brier_score', threshold=.5)
+
+    See also:
+        * properscoring.threshold_brier_score
+        * xskillscore.threshold_brier_score
+    """
+    if 'threshold' not in kwargs:
+        raise ValueError('Please provide threshold.')
+    else:
+        threshold = kwargs['threshold']
+    # switch args b/c xskillscore.threshold_brier_score(obs, forecasts)
+    return threshold_brier_score(reference, forecast, threshold)
+
+
+def _crps(forecast, reference, **kwargs):
     """
     Continuous Ranked Probability Score (CRPS) is the probabilistic MSE.
 
     Range:
         * perfect: 0
-        * max: 0
-        * else: negative
+        * min: 0
+        * max: ∞
```

Comment:
Hopefully, the infinity sign is supported everywhere

Reply:
seems to work with rtd. latex \inf didnt. riley checked some time ago